spec/lib/services/approved_itt_providers/update_spec.rb

```diff
@@ -0,0 +1,89 @@
+require "rails_helper"
+
+RSpec.describe Services::ApprovedIttProviders::Update do
+  let(:now) { "2022-12-12 18:00:00" }
+
+  around do |example|
+    Timecop.freeze(now) do
```

Comment:
We can do this with built in functionality now, we don't need Timecop fortunately. https://api.rubyonrails.org/v5.2.4/classes/ActiveSupport/Testing/TimeHelpers.html

This can be written as:
```
travel_to(Time.parse(now)) do
  example.run
end
```